src/main/java/duke/exception/EmptyByException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
```

Comment:
I think this is a good way to handle exceptions as the class name is self-explanatory. 👍